liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,500 @@
+const {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  ethers,
+  getEthersProvider,
+} = require('forta-agent');
+const BigNumber = require('bignumber.js');
+const config = require('../agent-config.json');
+const { getAbi, ts, callAPI } = require('./utils');
+
+// Stores information about each account
+const initializeData = {};
+
+// #region Global functions
+function createAlert(
+  developerAbbreviation,
+  protocolName,
+  protocolAbbreviation,
+  type,
+  severity,
+  borrowerAddress,
+  liquidationAmount,
+  shortfallAmount,
+) {
+  return Finding.fromObject({
+    name: `${protocolName} Compound Liquidation Threshold Alert`,
+    description: `The address ${borrowerAddress} has dropped below the liquidation threshold. `
+      + `The account may be liquidated for: $${liquidationAmount} USD`,
+    alertId: `${developerAbbreviation}-${protocolAbbreviation}-LIQUIDATION-THRESHOLD`,
+    type: FindingType[type],
+    severity: FindingSeverity[severity],
+    protocol: protocolName,
+    metadata: {
+      borrowerAddress,
+      liquidationAmount,
+      shortfallAmount,
+    },
+  });
+}
+
+async function verifyToken(data, tokenAddressImport) {
+  /* eslint-disable no-param-reassign */
+  const tokenAddress = tokenAddressImport.toLowerCase();
+  if (data.tokens[tokenAddress] === undefined) {
+    data.tokens[tokenAddress] = {};
+    data.tokens[tokenAddress].contract = new ethers.Contract(
+      tokenAddress,
+      data.cTokenABI,
+      data.provider,
+    );
+    const cContract = data.tokens[tokenAddress].contract;
+    data.tokens[tokenAddress].symbol = await cContract.symbol();
+    // cETH does not have an underlying contract, so peg it to wETH instead
```

Comment:
Nit: Insert an empty line to break up this large block of text.
```suggestion

    // cETH does not have an underlying contract, so peg it to wETH instead
```